### DISCUSSION TITLE
What does "This is to comply with Microsoft's removal of the stock Windows 10 taskbar in explorer.exe of said builds." mean exactly?

### DISCUSSION THREAD

**Discussion by Gunhedo** [valinet/ExplorerPatcher#3445] [General]
Hi,

Could one of the developers explain what "This is to comply with Microsoft's removal of the stock Windows 10 taskbar in explorer.exe of said builds." means exactly, please?

**Comment by pyrates999** [comment]
I think it means Microsoft removed all the code that was for the windows 10 taskbar, in windows 11 24h2's file explorer.exe.

**Comment by Amrsatrio** [comment]
They added a feature flag in 26002 to make the old taskbar code artificially crash explorer.exe when reached. We tried to "defuse that bomb." StartAllBack did that as well. Not long after, MS added EP and SAB into the program compatibility assistant (PCA) database to have them blocked from being run if file name and publisher are matching, as a punishment. So we had to give up offering the option to use the stock 10 taskbar on builds with that feature flag.

Now the code is totally gone on 26224+, so there's no way to "defuse that bomb" anymore. But don't worry, I've finished my taskbar reimplementation that will enter public beta testing soon.

**Reply by Gunhedo** [reply]
Thank you for explaining it in detail. I was always wondering what happened and why the taskbar couldn't be used anymore, like before! 👍

**Reply by mistertheloaf** [reply]
> Now the code is totally gone on 26224+, so there's no way to "defuse that bomb" anymore. But don't worry, I've finished my taskbar reimplementation that will enter public beta testing soon.

Thank you for working on this.  Much appreciated.

**Reply by gitoss** [reply]
> But don't worry, I've finished my taskbar reimplementation that will enter public beta testing soon.

The "Windows 10 (ExplorerPatcher)" taskbar is like early xmas, thank you - I hope you manage to keep it working in upcoming Win11 versions.

If it is a "reimplemtation" Microsoft shouln't have anything to complain about...
